client/app/pages/alert/components/MenuButton.jsx

```diff
@@ -23,9 +26,9 @@ export default function MenuButton({ doDelete, canEdit, mute, unmute, muted }) {
 
   const confirmDelete = useCallback(() => {
     Modal.confirm({
-      title: "Delete Alert",
-      content: "Are you sure you want to delete this alert?",
-      okText: "Delete",
+      title: t("Alerts:Delete Alert"),
+      content: t("Alerts:Are you sure you want to delete this alert?"),
```

Comment:
Since we've defined the `t` object as the translation form alerts, I feel we don't need this prefix here. Is it correct?

```suggestion
      title: t("Delete Alert"),
      content: t("Are you sure you want to delete this alert?"),
```

Reply:
This is related to the previous comment